import-automation/progress-dashboard-rest/app/resource/import_attempt.py

```diff
@@ -117,17 +121,17 @@ def get(self, attempt_id):
     def patch(self, attempt_id):
         """Modifies the value of a field of an existing attempt.
 
-        attempt_id is forbidden to modify.
+        It is not allowed to modify the value of attempt_id.
```

Comment:
Perhaps drop this comment in favor of "as a constant string" below.

Reply:
I was trying to that the ID of an import attempt could not be changed not the input

Reply:
Maybe "The attempt_id of an existing import attempt resource is forbidden to be patched."?